mitplan-frontend/src/components/Sheet/VerticalTimeline.tsx

```diff
@@ -19,7 +20,10 @@ const VerticalTimeline: React.FC<VerticalTimelineProps> = ({ mitplanId, sheetId
   const containerRef = useRef<HTMLDivElement>(null);
 
   const sheet = useSelector((state: RootState) => state.mitplans.mitplans[mitplanId]?.sheets[sheetId]);
-  const { assignmentEvents, encounter, columnCount, timeScale = 5 } = sheet || {};
+  const { assignmentEvents, encounterId, columnCount, timeScale = 5 } = sheet || {};
+  const encounter = allEncounters[encounterId];
```

Comment:
we get the encounter object here. either you can:
pass encounter.events to TimestampColumn
or pass encounter.id to TimestampColumn and rewrite that component to fetch the encounter object

Reply:
You said this was actually fine so leaving it unless otherwise Stated. Also clicking resolve conversation buttons just to see what they do. :^)